DEV-1/FinderOfMaxSequenceOfUniqueSymbols.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Linq;
+using System.Text;
+
+namespace DEV_1
+{
+    /// <summary>
+    /// This class is used to find the maximum number
+    /// of unique consecutive characters in a string.
+    /// </summary>
+    class FinderOfMaxSequenceOfUniqueSymbols
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                StringBuilder builderForStartString = new StringBuilder();
+                foreach (var a in args)
+                {
+                    builderForStartString.Append(a).Append(" ");
+                }
+               
+                string startString = builderForStartString.ToString();
```

Comment:
Why do you convert string to array? no need? you can simply iterate by string chars using []